src/server/headed-mode-plugin.js

```diff
@@ -0,0 +1,16 @@
+import { globSync } from 'glob';
```

Comment:
Looks like this line is failing CI ^

Reply:
Yeah, I don't know why. This looks similar to the issue I had last week that Max also [had yesterday](https://d2l.slack.com/archives/C0PHG3QB0/p1686851524778899)